src/commands/protocols/common/epoch-check-command.js

```diff
@@ -329,16 +329,17 @@ class EpochCheckCommand extends Command {
             hashFunctionId: agreement.hashFunctionId,
             epoch: agreement.currentEpoch,
             agreementId: agreement.agreementId,
+            assertionId: agreement.assertionId,
             stateIndex: agreement.stateIndex,
             closestNode: closestNode.index,
             leftNeighborhoodEdge: neighbourhoodEdges?.leftEdge.index,
             rightNeighborhoodEdge: neighbourhoodEdges?.rightEdge.index,
         };
 
         await this.commandExecutor.add({
-            name: 'submitCommitCommand',
-            sequence: [],
-            retries: COMMAND_RETRIES.SUBMIT_COMMIT,
+            name: 'simpleAssetSyncCommand',
```

Comment:
It might be that node has handled service agreement event before the data has been replicated on the network, and sync will fail.

Reply:
Initially we will handle this with delay for `getEligibleAgreementsForSubmitCommit` it is defined as `SERVICE_AGREEMENT_START_TIME_DELAY_FOR_COMMITS_SECONDS`